apidocs/classes/ContentGenerationAppSyncLambda.md

```diff
@@ -140,7 +140,7 @@ ___
 
 ### graphqlApi
 
-• `Readonly` **graphqlApi**: `IGraphqlApi`
+• `Readonly` **graphqlApi**: `GraphqlApi`
 
 Returns an instance of appsync.IGraphqlApi created by the construct
```

Comment:
```suggestion
Returns an instance of appsync.GraphqlApi created by the construct
```

Reply:
Yeah I changed IGraphqlApi to GraphqlApi to support graphqlurl property in CFN output.